modules/functions.js

```diff
@@ -108,30 +108,57 @@ module.exports = (client) => {
       command = client.commands.get(client.aliases.get(commandName));
     }
     if (!command) return `The command \`${commandName}\` doesn"t seem to exist, nor is it an alias. Try again!`;
-  
+
     if (command.shutdown) {
       await command.shutdown(client);
     }
     delete require.cache[require.resolve(`../commands/${commandName}.js`)];
     return false;
   };
 
+  client.loadBackgroundWorker = (workerName) => {
+    const worker = require(`../background/${workerName}`);
+    client.log("log", `Loading Background Worker: ${worker.conf.slug}. 👌`);
+    if (worker.init)
+      worker.init();
+    client.workers.set(worker.conf.slug, worker);
+    worker.intervalId = setInterval( (worker) => {
+      worker.run(client);
+    }, worker.conf.interval, worker);
+    return false;
+  };
+
+  client.unloadBackgroundWorker = (workerName) => {
+    let worker;
+    if (client.workers.has(workerName)) {
+      worker = client.workers.get(workerName);
+    }
+    if (!worker) return `The worker \`${workerName}\` doesn"t seem to exist. Try again!`;
```

Comment:
Since when do we use double inverted commas instead of apostrophes?

Reply:
Thats a york-dev requirement for codacy but guidebot has no rules for it